modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -20,6 +20,10 @@ ifeval::["{context}" == "installing-mirroring-disconnected"]
 :oc-mirror:
 endif::[]
 
+ifeval::["{context}" == "about-installing-oc-mirror-v2"]
+:oc-mirror-v2:
+endif::[]
```

Comment:
You need to add 3 lines at the end of the file to un-set the `:oc-mirror-v2:` variable:
```
ifeval::["{context}" == "about-installing-oc-mirror-v2"]
:!oc-mirror-v2:
endif::[]
```